catroid/src/main/java/org/catrobat/catroid/content/Project.java

```diff
@@ -310,16 +308,6 @@ public Scene getSceneByName(String name) {
 		return null;
 	}
 
-	public boolean manualScreenshotExists(String manualScreenshotName) {
-
-		String path = PathBuilder.buildProjectPath(getName()) + "/" + manualScreenshotName;
```

Comment:
since the screenshots are saved inside the Scene folder, this method will return always false, also it's not used anymore, therefore i deleted it